code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -13,6 +13,8 @@
 	var/hatching = 0 // So we don't spam ghosts.
 	var/datum/recruiter/recruiter = null
 	var/child_prefix_index = 1
+	var/last_ping_time = 0
+	var/ping_cooldown = 150
```

Comment:
15 seconds is a pretty long time, in that time someone might enter and exit the room without having seen it get pinged.